cpp/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@
 # For more information about CMake, see http://www.cmake.org             #
 ##########################################################################
 cmake_minimum_required(VERSION 3.17 FATAL_ERROR)
-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+#set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
```

Comment:
Do we need this?

Reply:
may be this was the problem. I added this to run examples without having to update `LD_LIBRARY_PATH`. It seems like we'll have to figure out an alternative to that

Reply:
I figured out the issue. It turned out that I had a built ucc with a different ucx version than the one in my conda env.